### ISSUE TITLE
[BUG] Getting timout when trying to run example alert

### ISSUE THREAD

**Issue by hoplers** [prometheus-msteams/prometheus-msteams#126]
I tried running the basic example on README, but im getting errors.

configuration -

docker run command - docker run -d -p 2000:2000 --name="promteams" -e TEAMS_INCOMING_WEBHOOK_URL="https://outlook.office.com/webhook/<xxxxxxx>" -e TEAMS_REQUEST_URI=alertmanager quay.io/prometheusmsteams/prometheus-msteams:v1.3.4

proxy configured (screenshot from docker inspect on image) -
![image](https://user-images.githubusercontent.com/29622398/82557999-7551de80-9b75-11ea-8883-4d3df1c0287b.png)

trying to run 'curl -v -X POST -d @prom-alert.json http://localhost:2000/alertmanager' as in the example. output is as following -
![image](https://user-images.githubusercontent.com/29622398/82558153-b8ac4d00-9b75-11ea-9539-764fb8f900b5.png)

logs from docker logs - 
![image](https://user-images.githubusercontent.com/29622398/82558508-7cc5b780-9b76-11ea-9108-d376d1fb9743.png)

the webhook works -  when I curl an office 365 card directly to the webhook it is showing in the channel.
any help will be appreciated. thank you.

**Comment by Knappek** [comment]
>  when I curl an office 365 card directly to the webhook it is showing in the channel

What exactly do you mean by this? If it is that you executed curl on your local computer, could you try to login to the prometheus-msteams docker container and execute curl from there (with setting the proxy env variables for your shell)? 

To me it looks like prometheus-msteams does not have access to the webhook. As you are routing through a proxy, you could try two things:
1. telnet on the proxy with port 911 and/or 912
2. execute a post request directly to MS teams from the proxy host (if you have access to it)

**Comment by hoplers** [comment]
Hey Andy,
Thanks for the quick response.
About your question - I ran the curl outside of the docker.
I tried running from inside like you said - didnt work, but after checking some things it seems that this is a DNS issues (and not a proxy issue). looks like something with our organizational DNS is not functioning properly from inside docker.
Anyway im pretty certain that this is an issue with my environment\configurations, and not issue with promteams, so i'm closing this issue.
Thanks!
